### ISSUE TITLE
@units

### ISSUE THREAD

**Issue by Dangleos** [Anuken/Mindustry-Suggestions#3051]
### Describe the content or mechanics you are proposing.

Add constant @units in processor that shows how much units is binded

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

If there is @links why is there no @units.
It would help binding a certain amount of units in just one processor

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by MHeMoTexHuK** [comment]
> If there is @links why is there no @Units.

Because
A processor can't have more than 1 unit bount at the same time.

**Comment by AmosNico** [comment]
> > If there is @links why is there no @Units.
> 
> Because A processor can't have more than 1 unit bound to it.

True, although it can have multiple units that are controlled by it, so I think that is what the author means.

**Comment by SMOLKEYS** [comment]
> certain amount of units

Dude, use unit flags.
